src/test/java/ancaMarian/tests/tema6/HerokuForms.java

```diff
@@ -0,0 +1,144 @@
+package ancaMarian.tests.tema6;
+
+import ancaMarian.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import teofilursan.driver.BrowserManager;
+
+public class HerokuForms {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws InterruptedException {
+        navigateToHerokuHomePage();
+        maximizeWindow(driver);
+        navigateToHerokuFormsPage();
+        inputUsername();
+        inputPassword();
+        inputComment();
+        addFile();
+        selectCheckbox();
+        deselectCheckbox();
+        scrollToTheBottomOfThePage();
+        selectRadioItem();
+        selectFromMultipleValues();
+        selectFromDropdownList();
+        scrollToSubmitButton();
+        FileUtils.takeScreenshot(driver, "form");
+        goBackToFormPage();
+        try {
+            goBackToFormPage();
+        } catch (NoSuchElementException e) {
+            FileUtils.takeScreenshot(driver, "ElementNotFound");
+            System.out.println("Nu s-a gasit elementul pe pagina. S-a salvat un screenshot cu denumirea ElementNotFound.");
+        } finally {
+            closeBrowser();}
+    }
+
+    public static void navigateToHerokuHomePage(){
+        driver = ancaMarian.driver.BrowserManager.createChromeDriver();
```

Comment:
Replace BrowserManager.createChromeDriver() with BrowserManager.createDriver() - the method from your package. Thanks!